Initial Constitution

silky opened the pull request:
Related to #9

Related to #2 

I'm just following this exact example - https://static1.squarespace.com/static/5f9323e965b0950d66514484/t/5fbaf666e8de8f008859aef2/1606088294991/Protect+Earth+Constitution+-+Approved+171120.pdf

silky committed:
more doc building

silky committed:
wip constitution; auto-dating.

silky committed:
makefile cleanup

silky committed:
wip

silky committed:
import parts of the constitution

silky committed:
sloooooow progress.

silky committed:
more text and first trustees

silky committed:
more writing

silky committed:
more sections

silky committed:
wip

silky committed:
first pass is complete

silky committed:
her/him -> they

silky commented:
This is "officially" complete, as a first draft.

Honestly, I think this document is not really considered ready until it's somehow legally approved by the home office and/or the scottish charity thing.

**BUT**, it is important for you @marcosh and @gacafe to have a glance at it.

Notably, the `boilerplate.md` is copied almost literally from the reference PDF; aside from some small (not important) corrections.

The only parts that are even different are the actual file `README.md` because it contains our `Objects` (i.e. our "Mission").

**This could well be revised**, and certainly your input (and, to anyone reading this, your input is also valued!)

And, of course, the `first-trustees.md`. the `minutes.md` is unchanged, but i thought i'd pop it out incase we decided to change it.

Anyway, please take a glance at whatever you like.

**Of course I won't actually submit this to the government without checking with you directly first**. But anyway, I do want to try and use the PR process to make changes :)

Note: If you want to run and view the PDF yourself, you can run `make watch-pdfs` from within the nix shell, which will rebuild it on change.

Once we're happy with a final version, I think we'll commit the actual PDF as well.

silky commented:
Note that one aspect that comes out of this is that, essentially, the spending decisions come to down to the *trustees*; (i.e. us 3) and not the "commtitee". I think this is fine; we can just have the informal understanding that we will always follow the committee's recommendations; this saves us formalising that process.

I'm open to thoughts on this, though.

I do think, that, annoyingly, because this does _not_ fix #9, we'll need to call the document that _does_ do that something else; which is fine, but as above it means that it's more like "operating guidance" than anything legal.)

Again, I think this is totally fine because we're all acting in good faith; but maybe there's something to be cautious about there?

I'd be tempted to leave those things slightly informal for the first 3-6 months, say, and then if things change and end up being successful, we could push on with some kind of formalisation. As usual, I don't really want to drown in overwhelming paperwork initially.

As with all these rambling comments; quite open to opinions.

silky commented:
sorry and I should just say I've done this under the assumption that at some point the application for a CIO involves uploading this document for review.

one thing i was told is that it helps to follow a simple template; that's why i copied this (from the person that told me that!) in the hopes that that's how it works 😅 

it could well not be the case, once we start to investigate _actually_ how to apply, which I'll try and do next week.

marcosh commented on constitution/README.md:
```diff
@@ -1 +1,44 @@
+>     a. Is innovative in the industry, or
+>     b. Provides value to the local community, or
+>     c. Addresses marginalised communities, or
+>     d. Provides value to the individual themselves, or
```
Are we sure we would like to fund projects valuable only for a single individual?

silky replied:
@marcosh Yeah, i wasn't quite sure how to word this part.

What I had in mind was, suppose a project comes in that doesn't really add anything new, _but_ is a growth opportunity for the specific person. I kind had in mind some of the ideas in this blog post - https://jvns.ca/blog/2017/02/16/hiring---opportunity/ 

i.e. is the opportunity _good for the person_?

But maybe, upon reflection, this doesn't quite make sense? Very happy to re-word as you see fit. I think it's worth nit-picking this, because I think we'll come back to reference it quite frequently.

silky replied:
@gacafe any wording suggestions you may have would also be welcome :)

marcosh replied:
It makes sense to me if the growth for the person is the first step of a bigger plan which has more broader impact.

I'm not necessarily saying that self-scoped projects are not good, but it might make sense, at least in the beginning, try to focus on project which have a little broader impact

silky replied:
i've attempted a minor re-wording, maybe that helps @marcosh ?

@gacafe should also be able to take a look today, hopefully, or at least this week.

marcosh replied:
I like this, nice work @silky!

marcosh commented on constitution/boilerplate.md:
```diff
@@ -0,0 +1,686 @@
+>    in the General Regulations and in particular:
+>
+>    a. The requirement to provide within 21 days to any member on request a hard
+>       cop of any document or information sent to the member otherwise than in hard
```
```suggestion
>       copy of any document or information sent to the member otherwise than in hard
```

marcosh reviewed (approved):
looks good to me!

silky committed:
Update constitution/boilerplate.md

Co-authored-by: Marco Perone <pasafama@gmail.com>

Update constitution/README.md

Co-authored-by: Nicolas Jeannerod <niols@niols.fr>

Update constitution/boilerplate.md

Co-authored-by: Nicolas Jeannerod <niols@niols.fr>

Update constitution/boilerplate.md

Co-authored-by: Nicolas Jeannerod <niols@niols.fr>

Update constitution/boilerplate.md

Co-authored-by: Nicolas Jeannerod <niols@niols.fr>

Update constitution/boilerplate.md

Co-authored-by: Nicolas Jeannerod <niols@niols.fr>

Update constitution/boilerplate.md

Co-authored-by: Nicolas Jeannerod <niols@niols.fr>

Update constitution/boilerplate.md

Co-authored-by: Nicolas Jeannerod <niols@niols.fr>

Update constitution/boilerplate.md

Co-authored-by: Nicolas Jeannerod <niols@niols.fr>

Apply suggestions from code review

Co-authored-by: Nicolas Jeannerod <niols@niols.fr>

Niols commented on constitution/README.md:
```diff
@@ -1 +1,44 @@
+> 1. For the public benefit, build and support a community around sustainable
+>    climate technology projects, both individual and commercial; and
+>
+> 2. For the public benefit, fund and distribute a no-strings attached grant
```
```suggestion
> 2. For the public benefit, fund and distribute a no-strings-attached grant
```

Niols commented on constitution/boilerplate.md:
````diff
@@ -0,0 +1,686 @@
+>     first-trustees.md
+>     ```
+
+10. **Appointment of charity trustees**
````
Up to 9, the title numbers all finished by `)`. So here is a batch of corrections for all the other ones. Feel free to disregard this, obviously.

```suggestion
10.) **Appointment of charity trustees**
```

Niols commented on constitution/boilerplate.md:
```diff
@@ -0,0 +1,686 @@
+>    trustees must have regard to the skills, knowledge, and experience needed
+>    for the effective administration of the CIO.
+
+11. **Information for new charity trustees**
```
```suggestion
11.) **Information for new charity trustees**
```

Niols commented on constitution/boilerplate.md:
```diff
@@ -0,0 +1,686 @@
+>   b. A copy of the CIO's latest Trustee's Annual Report and statement of
+>      accounts.
+
+12. **Retirement and removal of charity trustees**
```
```suggestion
12.) **Retirement and removal of charity trustees**
```

Niols commented on constitution/boilerplate.md:
```diff
@@ -0,0 +1,686 @@
+>
+> 2. Any person retiring as a charity trustee is eligible for reappointment.
+
+13. **Taking of decisions by charity trustees**
```
```suggestion
13.) **Taking of decisions by charity trustees**
```

Niols commented on constitution/boilerplate.md:
```diff
@@ -0,0 +1,686 @@
+>       charity trustees have previously resolved, and delivered to the CIO at
+>       its principal office or such other place as the trustees may resolve.
+
+14. **Delegation by charity trustees**
```
```suggestion
14.) **Delegation by charity trustees**
```

Niols commented on constitution/boilerplate.md:
```diff
@@ -0,0 +1,686 @@
+>    c. The charity trustees shall from time to time review the arrangements
+>       which they have made for the delegation of their powers.
+
+15. **Meetings of charity trustees**
```
```suggestion
15.) **Meetings of charity trustees**
```

Niols commented on constitution/cio-membership.md:
```diff
@@ -0,0 +1,14 @@
+  but it could be reconsidered another time.
+---
+
+16. **Membership of the CIO**
```
```suggestion
16.) **Membership of the CIO**
```

Niols commented on constitution/boilerplate.md:
```diff
@@ -0,0 +1,686 @@
+> 5. If the CIO is to be wound up or dissolved in any other circumstances, the
+>    provisions of the Dissolution Regulations must be followed.
+
+30. **Interpretation**
```
```suggestion
30.) **Interpretation**
```

Niols commented on constitution/boilerplate.md:
```diff
@@ -0,0 +1,686 @@
+>    the, and the amendment does not take effect until it has been recorded in
+>    the Register of Charities.
+
+29. **Voluntary winding up or dissolution**
```
```suggestion
29.) **Voluntary winding up or dissolution**
```

Niols commented on constitution/boilerplate.md:
```diff
@@ -0,0 +1,686 @@
+> first try in good faith to settle the dispure my mediation before resorting
+> to litigation.
+
+28. **Amendment of constitution**
```
```suggestion
28.) **Amendment of constitution**
```

Niols commented on constitution/boilerplate.md:
```diff
@@ -0,0 +1,686 @@
+> rules or by-laws currently in force must be made available to any member of
+> the CIO on request.
+
+27. **Disputes**
```
```suggestion
27.) **Disputes**
```

Niols commented on constitution/boilerplate.md:
```diff
@@ -0,0 +1,686 @@
+>    Commission within 28 days of any change in the particulars of the CIO
+>    entered on the Central Register of Charities.
+
+26. **Rules**
```
```suggestion
26.) **Rules**
```

Niols commented on constitution/boilerplate.md:
````diff
@@ -0,0 +1,686 @@
+minutes.md
+```
+
+25. **Accounting records, accounts, annual reports and returns, register maintenance**
````
```suggestion
25.) **Accounting records, accounts, annual reports and returns, register maintenance**
```

Niols commented on constitution/boilerplate.md:
```diff
@@ -0,0 +1,686 @@
+> relation to the keeping of, and provision of access to, a register of its
+> members and charity trustees.
+
+24. **Minutes**
```
```suggestion
24.) **Minutes**
```

Niols commented on constitution/boilerplate.md:
```diff
@@ -0,0 +1,686 @@
+>           or charity trustee who has not consented to receive communications in
+>           electronic form.
+
+23. **Keeping of Registers**
```
Careful when batch-accepting (or rejecting!) those; this one also puts the “R” of “registers” lowercase, as there does not seem to be any capitalisation of the titles in this document.

```suggestion
23.) **Keeping of registers**
```

Niols commented on constitution/boilerplate.md:
```diff
@@ -0,0 +1,686 @@
+>       document to which the seal is affixed and unless otherwise so
+>       determined it shall be signed by two charity trustees.
+
+22. **Use of electronic communications**
```
```suggestion
22.) **Use of electronic communications**
```

Niols commented on constitution/boilerplate.md:
```diff
@@ -0,0 +1,686 @@
+>    sub-clause (1), the resolution would have been void, or if the charity
+>    trustee has not compiled with clause 7 (Conflicts of interest).
+
+21. **Execution of documents**
```
```suggestion
21.) **Execution of documents**
```

Niols commented on constitution/boilerplate.md:
```diff
@@ -0,0 +1,686 @@
+> apply to any general meeting of the members, with all references to
+> trustees to be taken as references to members.
+
+20. **Saving provisions**
```
```suggestion
20.) **Saving provisions**
```

Niols commented on constitution/boilerplate.md:
```diff
@@ -0,0 +1,686 @@
+> is limited to members who are members of the CIO on the date when the
+> proposal is first circulated.
+
+19. **General meetings of members**
```
```suggestion
19.) **General meetings of members**
```

Niols commented on constitution/boilerplate.md:
```diff
@@ -0,0 +1,686 @@
+>    members for any purpose under the Charities Acts, General Regulations or
+>    Dissolution Regulations.
+
+18. **Decisions which  must be made by the members of the CIO**
```
```suggestion
18.) **Decisions which  must be made by the members of the CIO**
```

Niols commented on constitution/boilerplate.md:
````diff
@@ -0,0 +1,686 @@
+cio-membership.md
+```
+
+17. **Informal or associate (non-voting) membership**
````
```suggestion
17.) **Informal or associate (non-voting) membership**
```

Niols reviewed (approved):
LGTM

silky committed:
main statement re-wording

silky committed:
sign commits

silky committed:
essential criteria to it's own file for easy referencing

silky commented:
Okay, I'm going to merge this.

It doesn't mean that this is the final version that's submitted; but I think it's been long enough now. I'll review the essential parts with @gacafe offline; and if there's any changes I'll put up another PR.

I'll make one change which is to commit the finalised PDF; just for easy consumability.

silky committed:
made the README reference the pdf; initial pdf

silky committed:
working link

silky committed:
formatting
